Sources/Datadog/Logger.swift

```diff
@@ -72,53 +78,53 @@ public class Logger {
     ///   - message: the message to be logged
     ///   - attributes: a dictionary of attributes to add for this message. If an attribute with
     /// the same key already exist in this logger, it will be overridden (just for this message).
-    public func debug(_ message: String, attributes: [AttributeKey: AttributeValue]? = nil) {
-        log(level: .debug, message: message, messageAttributes: attributes)
+    public func debug(_ message: String, error: Error? = nil, attributes: [AttributeKey: AttributeValue]? = nil) {
```

Comment:
All this `public` APIs changed in `Logger` need the documentation comment update.